data/json/mapgen/necropolis.json

```diff
@@ -728,13 +728,13 @@
             "place_monsters": [
                 { "monster": "GROUP_NECROPOLIS", "x": [ 1, 22 ], "y": [ 1, 22 ], "density": 0.10 }
             ],
-            "place_items": [
-                { "item": "building_rubble", "chance": 90, "x": [ 21,23 ], "y": [ 17,20 ]},
-                { "item": "building_rubble", "chance": 90, "x": [ 11,20 ], "y": [ 19,21 ]},
-                { "item": "building_rubble", "chance": 90, "x": [ 16,17 ], "y": [ 17,18 ]},
-                { "item": "building_rubble", "chance": 90, "x": [ 5,10 ], "y": [ 17,18 ]},
-                { "item": "home_hw", "chance": 40, "x": [ 12,22 ], "y": [ 8,8 ]},
-                { "item": "softdrugs", "chance": 50, "x": [ 10,10 ], "y": [ 13,15 ]}
+            "place_loot": [
```

Comment:
Can you give me an example of a situation where moving from one to the other may cause a problem? I'd like to avoid causing the game to explode or violate intended balance.